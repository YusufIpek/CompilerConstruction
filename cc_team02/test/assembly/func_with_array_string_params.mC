void do_something_with_arrays(string[10] a, int index, string value){
	a[index]=value;
}

void main(){
	string[10] arr;
	int index;
	string value;
	index=4;
	value="999.99+0.1";

	do_something_with_arrays(arr,index,value);
	
	print(arr[index]);
}