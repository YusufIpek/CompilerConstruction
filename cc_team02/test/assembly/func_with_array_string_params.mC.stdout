999.99+0.1